src/components/atoms/Icon/Icon.js

```diff
@@ -1,19 +1,43 @@
+import styles from '!!raw-loader!./Icon.css';
+import varStyles from '!!raw-loader!../../../shared/variables.css';
+import bootstrapStyles from '!!raw-loader!../../../shared/themed-bootstrap.css';
+
+
 export default class Icon extends HTMLElement {
   constructor() {
     // Always call super first in constructor
     super();
-    // Create a shadow root.
-    this.attachShadow({ mode: 'open' });
+   // Create a shadow root
+  const shadow = this.attachShadow({ mode: 'open' });
+
+        // Add styles
+        const bootStyles = document.createElement('style');
+        bootStyles.textContent = bootstrapStyles;
+        const variableStyles = document.createElement('style');
+        variableStyles.textContent = varStyles;
+        const itemStyles = document.createElement('style');
+        itemStyles.textContent = styles;
+        shadow.appendChild(bootStyles);
+        shadow.appendChild(variableStyles);
+        shadow.appendChild(itemStyles);
   }
 
   connectedCallback() {
+    // Boolean Attribute adds circle if present
+    const isHighlighted = this.hasAttribute('is-highlighted');
+
```

Comment:
Line 29 checks if all this code has already been executed and prevents the icon from being recreated if so (for performance and presentation reasons).

So this line should definitely be below line 31. It'd probably be best placed on line 68. It's a good practice to collocate variable definitions with where the variable is used.